src/style.css

```diff
@@ -1 +1,3 @@
-/* styles go here */
+span {
```

Comment:
this task could be done without extra span tag, try to use only that elements you really need to display the desired page